### PR TITLE
c2

### PR DESCRIPTION
**Pull request by merizamed**
desc c2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 aa
 bb
-
+cc
```
Comment (silverqx) [inline README.md:5]:
```suggestion
ccc
```

**Review by silverqx — CHANGES_REQUESTED** [review]
req1

File: README.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 aa
 bb
-
+ccc
```
Comment (silverqx) [inline README.md:5]: not good cccc

**Review by silverqx — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (merizamed) [reply README.md:5]: Ok i'm gona post it.

Comment (merizamed) [reply README.md]: done

**Review by silverqx — CHANGES_REQUESTED** [review]
please add also dd

**Review by silverqx — CHANGES_REQUESTED** [review]
please add also ee

**Comment by merizamed** [comment]
Ok, i'm gona do it, ee

**Review by silverqx — CHANGES_REQUESTED** [review]
also ff

File: README.md [hunk 3]
```diff
@@ -2,4 +2,6 @@
 
 aa
 bb
-
+cccc
+dd
+ee
```
Comment (silverqx) [inline README.md:7]: another ff

**Review by silverqx — APPROVED** [review]
ok, good to me approving

**Pull request merged**